app/views/teaching_events/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="teaching-events">
+  <h1>Teaching events</h1>
```

Comment:
```suggestion
  <h1>Get into Teaching events</h1>
```
We know from UR that people find 'Teaching events' hard to understand - are they for teachers?

Suggest we try this instead for now.

Should/could this be styled in the same way that the blog h1 is styled (numberplate!)?